app/domain/payments/ebics_import.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+#  Copyright (c) 2021, Die Mitte Schweiz. This file is part of
+#  hitobito and licensed under the Affero General Public License version 3
+#  or later. See the COPYING file at the top-level directory or at
+#  https://github.com/hitobito/hitobito.
+
+class Payments::EbicsImport
+  def initialize(payment_provider_config)
+    @payment_provider_config = payment_provider_config
+  end
+
+  def run
+    return [] unless @payment_provider_config.pending? || @payment_provider_config.registered?
+
+    created_payments
+  end
+
+  private
+
+  def created_payments
+    payment_provider = PaymentProvider.new(@payment_provider_config)
+
+    payment_provider.HPB
```

Comment:
Und wenn sich die Keys dann doch geändert haben, wird es eine Entwickleraufgabe, richtig?

Mehr zum Verständnis, ich bin gespannt, wie sich das entwickelt …

Ich überlege, ob man da was dokumentieren muss. Aktuell ist es ja noch frisch im Kopf.

Reply:
Ja wenn die Keys nicht mit denjenigen in den Settings übereinstimmen wird ein Error geraised.
Dann müsste man als Entwickler die entsprechenden Werte in den Settings updaten. Ich kann das sonst gerne noch wo dokumentieren. Grad die Frage ob wiki oder Github aber eigentlich schadet es ja nicht wenns direkt im Github ist. Dann kann das auch direkt zum restlichen feature doc

Reply:
Das fände ich gut, merci.